Day-II-Calculator/calculator/src/App.js

```diff
@@ -1,22 +1,14 @@
 import React from 'react';
 import './App.css';
+import Buttons from './components/ButtonComponents/Buttons';
+import CalculatorDisplay from './components/DisplayComponents/CalculatorDisplay';
 
 const App = () => {
-  return (
-    <div>
-      <h3>Welcome to React Calculator</h3>
-      <p>
-        We have given you a starter project. You'll want to build out your
-        components in their respective files, remove this code and replace it
-        with the proper components.
-      </p>
-      <p>
-        <strong>
-          Don't forget to `default export` your components and import them here
-          inside of this file in order to make them work.
-        </strong>
-      </p>
-    </div>
+    return (
+        <div className="container">
+            <CalculatorDisplay />
+            <Buttons />
+        </div>
```

Comment:
See if you can find a way to make the calculator font match a real calculator. (HINT: 14 segment display works best to show the multiplication symbol best.)